tests/calc/test_indices.py

```diff
@@ -130,16 +131,15 @@ def test_weighted_continuous_average():
     assert_almost_equal(v, 6.900543760612305 * units('m/s'), 7)
 
 
-@pytest.mark.xfail(reason='hgrecco/pint#1593')
+@pytest.mark.xfail(condition=module_version_before('pint', '0.21'), reason='hgrecco/pint#1593')
```

Comment:
I wonder what it would take to make this: `version_check('pint<0.21')`

I'm not saying we have to go that route or that it's better, but my brain has now twice asked me the question so I at least had to post to get it out of there.

Reply:
I'll give it a shot!

Reply:
Alright, take a look.